docs/cloud/cloud-databases/cloud-db-serverless-scale.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Scale a serverless database
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 3
+---
+
+# How do I scale a serverless up or down database?
+{: .no_toc }
+
+{% include /serverless/serverless-preview-warning.md %}
+
+This procedure explains how to scale a serverless database up or down. 
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+{% include /serverless/serverless-worker-shape.md %}
+{% include /serverless/serverless-create-defaults.md %}
```

Comment:
These two includes go under an "Additional information" heading that would sit between the syntax and the example